.eslintrc.json

```diff
@@ -16,5 +16,23 @@
   "rules": {
     "padding-line-between-statements": "off",
     "react/jsx-uses-vars": "error"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "project": "./tsconfig.json"
+      },
+      "extends": [
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/recommended-requiring-type-checking"
```

Comment:
@calebeby In the [testingjavascript.com](https://testingjavascript.com/) **Static Analysis Testing JavaScript Applications** module, Kent adds the [`eslint-config-prettier/@typescript-eslint` plugin](https://github.com/prettier/eslint-config-prettier). Does it make sense to add here?  🤔 

https://github.com/kentcdodds/static-testing-tools/blob/34d73e3251a0dd8204b2629b27b41659fd8f054d/.eslintrc#L17

I feel a bit over my element so I'm mostly curious because I don't understand. 🙂

Reply:
Added :heavy_check_mark: